sdk/cognitivelanguage/azure-ai-language-questionanswering/azure/ai/language/questionanswering/aio/operations/_question_answering_client_operations.py

```diff
@@ -216,6 +253,30 @@ async def query_text(
     async def query_text(
         self, *args, **kwargs: Any
     ) -> "_models.TextAnswers":
+        """Answers the specified question using the provided text in the body.
+
+        :param text_query_options: Post body of the request. Provide either `text_query_options`, OR
+         individual keyword arguments. If both are provided, only the options object will be used.
+        :type text_query_options: ~azure.ai.language.questionanswering.models.TextQueryOptions
+        :keyword question: User question to query against the given text records. Provide either `text_query_options`, OR
+         individual keyword arguments. If both are provided, only the options object will be used.
+        :paramtype question: str
+        :keyword records: Text records to be searched for given question. Provide either `text_query_options`, OR
+         individual keyword arguments. If both are provided, only the options object will be used.
+        :paramtype records: list[~azure.ai.language.questionanswering.models.TextRecord]
```

Comment:
I take it these can't also just be strings, right? From the implementation I don't see any auto-incrementing like Text Analytics did, which I copied for .NET.

Probably don't need to worry about it now, but you might open an issue to consider for beta 2.

Reply:
Yeah I will open an issue for this.